catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,6 +28,21 @@
         <property name="indexLocation" value="data/geonames-index"/>
     </bean>
 
+    <reference id="geoExtractor" interface="org.codice.ddf.spatial.geocoding.GeoEntryExtractor"/>
+
+    <bean id="executorService" class="java.util.concurrent.Executors"
+          factory-method="newSingleThreadExecutor">
+    </bean>
+
+    <bean id="indexInitializer" class="org.codice.ddf.spatial.geocoding.index.IndexInitializer"
+          init-method="init" destroy-method="destroy">
+        <property name="indexLocationPath" value="data/geonames-index"/>
+        <property name="defaultGeonamesDataPath" value="data/default_geonames_data.zip"/>
```

Comment:
❓  will there be any issue with these paths on other systems like Windows?

Reply:
Instead of hard coding the character here we could use the extras parameters on `Path.get(String, String...)` so that it builds the path for us

Reply:
I will double check that this will work on windows. It is following the same pattern already used in the file right now but that doesn't make it right